### ISSUE TITLE
Add grow parameter to clipping

### ISSUE THREAD

**Issue by janerigby** [astropy/ccdproc#753]
A feature request:  one very helpful feature that iraf's imcombine had, that I believe is missing from ccdproc, is the ability to "grow" a masked region spatially.  astropy.stats.sigma_clip() does have this ability, via the "grow" parameter.  One basic use case is to mask all sources in each twilight flat image before median combining, in order to make a flat-field.

**Comment by mwcraig** [comment]
@janerigby -- version 2.2.0, released yesterday, adds an option to use astropy's `sigma_clip` for sigma clipping in ccdproc, which means you can also use the grow parameter. 

I'm not planning to mention the grow parameter in the release notes because I haven't had a chance to test it, but using it would look like this:

```python
c = Combiner(your_images)
c.sigma_clipping(low_thresh=3, high_thresh=3, use_astropy=True, grow=1)
```

**Comment by mwcraig** [comment]
Since ccdproc now uses astropy's `sigma_clip` by default I'm going to close this. Feel free to reopen if appropriate (and congrats on the outstanding stuff you have done with JWST).

Closed by #794
